src/portal/lib/src/project-policy-config/project-policy-config.component.html

```diff
@@ -20,8 +20,8 @@
             <clr-control-helper class="config-subtext"> {{ 'PROJECT_CONFIG.CONTENT_TRUST_POLCIY' | translate }}
             </clr-control-helper>
         </clr-checkbox-container>
-        <clr-checkbox-container id="prevent-vulenrability-image">
-            <label><span>{{ 'PROJECT_CONFIG.SECURITY' | translate }}</span></label>
+        <clr-checkbox-container id="prevent-vulenrability-image" class="margin-top-05">
+            <label></label>
```

Comment:
I feel that the way to delete Clair was rude. The bug here is because Clair and notory decided whether to display it. I think if there is no scanner in this place, we should discuss it